docs/plone-deployment/edit-project.md

```diff
@@ -97,30 +123,32 @@ default_context:
     db_storage: direct
 ```
 
-To activate the settings run `make build-backend` in the root folder of the project.
+To activate the settings, run `make build-backend` in the root folder of the project.
+
 
-## Frontend
+### Frontend
 
-Go to folder `frontend` and edit the package.json.
+Go to the folder `frontend`, and edit the file `package.json`.
 
-Append `@plone-collective/volto-authomatic` to the lists of `dependencies` and `addons`.
+Append `@plone-collective/volto-authomatic` to the lists `dependencies` and `addons`.
 
-To activate the settings run `make build-frontend` in the root folder of the project.
+To activate the settings, run `make build-frontend` in the root folder of the project.
 
-## Activate the plugin
 
-You need to login as administrator to Plone Backend (in Classic UI) at `http://localhost:8080`.
+### Activate the plugin
 
-At the bottom of the menu on the left choose `admin`, then `Site Setup` and then `add-ons`.
+You need to login to Plone Backend as Administrator (in Classic UI) at `http://localhost:8080`.
+
+At the bottom of the menu on the left, choose {guilabel}`admin`, {guilabel}`Site Setup`, then {guilabel}`add-ons`.
```

Comment:
For my own education: what does using `{guilabel}` here accomplish?

Reply:
It provides semantic markup that renders as shown in the build preview: https://deploy-preview-691--plone-training.netlify.app/plone-deployment/edit-project.html#activate-the-plugin

It is used for menus, clickable items, and other UI items, but not keyboard which is `{kbd}`.